modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusStateUpdater.scala

```diff
@@ -338,6 +348,35 @@ object ConsensusStateUpdater {
     private def pickMajority[A: Order](proposals: List[A]): Option[A] =
       proposals.foldMap(a => Map(a -> 1)).toList.map(_.swap).maximumOption.map(_._2)
 
+    private def pickValidatedMajority(
+      lastSignedArtifact: Signed[Artifact],
+      trigger: ConsensusTrigger,
+      resources: ConsensusResources[Artifact]
+    )(proposals: List[Hash]): F[Option[Hash]] =
+      proposals
+        .foldMap(a => Map(a -> 1))
+        .toList
+        .map(_.swap)
+        .sortBy { case (occurrences, hash) => (-occurrences, hash) }
```

Comment:
Still, invalid sorting on the 2nd param (not the same as in `pickMajority`). This should work

```
.toList
.map(_.swap)
.sorted
.reverse
```

Reply:
It doesn't matter which one we choose until we choose the same on all nodes.